lib/Parser/Constants.php

```diff
@@ -70,13 +70,17 @@ abstract class Constants
 
     const VK_ALL_DRIVERS = [
         "anv",
+        "lvp",
         "radv",
+        "tu",
         "v3dv",
     ];
 
     const VK_ALL_DRIVERS_VENDORS = [
         "Intel"     => [ "anv" ],
+        "Lavapipe"  => [ "lvp" ],
```

Comment:
I'd put `lvp` under the `Software` vendor (as for OpenGL and `llvmpipe`)
```suggestion
        "Software"  => [ "lvp" ],
```